cognite/powerops/resync/data_classes.py

```diff
@@ -11,7 +11,7 @@
 
 from cognite.powerops.utils.serialization import load_yaml
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger("rich")
```

Comment:
**question**: What is this doing (and why)?

Reply:
OK. I don't really have an opinion about what's best - just wanted to know the reason for the change.